src/scripts/main.js

```diff
@@ -1,7 +1,32 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.className = 'notification';
+  message.classList.add(type);
+
+  const messageTitle = document.createElement('h2');
+
+  messageTitle.className = 'title';
+  messageTitle.textContent = title;
+
+  const messageDescription = document.createElement('p');
+
+  messageDescription.textContent = description;
+
+  message.append(messageTitle, messageDescription);
+  document.body.append(message);
+
+  message.style.position = 'absolute';
+
+  message.style.left =
+    document.offSetWidth - posRight - message.clientWidth + 'px';
```

Comment:
There is a typo in `document.offSetWidth`. It should be `document.documentElement.offsetWidth` or `window.innerWidth` to get the correct width of the document. Additionally, ensure that `message.clientWidth` is subtracted from the correct width to position the notification correctly.